src/base/bittorrent/torrenthandle.h

```diff
@@ -155,6 +155,33 @@ namespace BitTorrent
         Error
     };
 
+    class FilePriority
```

Comment:
Very peculiar use of the class... You just use it as a namespace. I don't think that makes any sense.

Reply:
So then I'll go back to having an explicit namespace for this. An enum class alone doesn't satisfy my requirements of having an `isValidPriority()` method.

Reply:
This has been implemented as an enum class now, with `isValidFilePriority()` defined in an anonymous namespace inside `torrentscontroller.cpp`.